localization/yabloc/yabloc_monitor/launch/yabloc_monitor.launch.xml

```diff
@@ -0,0 +1,8 @@
+<launch>
+  <arg name="param_file" default="$(find-pkg-share yabloc_monitor)/config/yabloc_monitor.param.yaml"/>
+
+  <node name="yabloc_monitor" pkg="yabloc_monitor" exec="yabloc_monitor" output="screen">
+    <param from="$(var param_file)"/>
+    <remap from="~/input/yabloc_pose" to="/localization/pose_estimator/pf/pose"/>
```

Comment:
I think this remapping is wrong :thinking: 
```suggestion
    <remap from="~/input/yabloc_pose" to="/localization/pose_estimator/yabloc/pf/pose"/>
```

Reply:
Right, forgot to reflect the latest one. Thanks!

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/4395/commits/b9cf1ae353f199dbb275a482ffb53e005211a90f